index.d.ts

```diff
@@ -718,7 +718,10 @@ declare module 'native-base' {
     /**
      * NativeBase.Drawer
      */
-    export class Drawer extends React.Component<NativeBase.Drawer, any> { }
+    export class Drawer extends React.Component<NativeBase.Drawer, any> { 
+        close: Function;
+        open: Function;
+    }
```

Comment:
Does the native-base Drawer have open()/close() methods? I know the `_root` does.
btw, there is a prop called "open: boolean" from underneath 'react-native-drawer', which actually controls the open/close of the drawer.